services/common/rs_server_common/utils/utils.py

```diff
@@ -502,6 +502,9 @@ def create_stac_collection(
         product_data = extract_eo_product(product, stac_mapper)
         feature_tmp = odata_to_stac(copy.deepcopy(feature_template), product_data, stac_mapper)
         item = stac_pydantic.Item(**feature_tmp)
+        # Add a default bbox and geometry, since L0 chunks items are not geo-located.
+        item.bbox = (-10.0, 50.0, -5.0, 55.0)
```

Comment:
Could you instead define a default bbox and geometry on the whole world so it's obvious that is is fake ? 
```
bbox = [-180.0, -90.0, 180.0, 90.0]
coordinates = [[[-180, -90], [180, -90], [180, 90], [-180, 90], [-180, -90]]]
```

Reply:
Ok, tests will also have to be changed